cl_dll/vgui_ScorePanel.h

```diff
@@ -149,26 +150,25 @@ class CLabelHeader : public Label
 		_dualImage->GetImage(0)->setText(text);
 
 		// calculate the text size
-		Font *font = _dualImage->GetImage(0)->getFont();
-		_gap = 0;
-		for (const char *ch = text; *ch != 0; ch++)
-		{
-			int a, b, c;
-			font->getCharABCwide(*ch, a, b, c);
-			_gap += (a + b + c);
-		}
-
-		_gap += XRES(5);
+		int wide, tall;
+		_dualImage->GetImage(0)->getTextSize(wide, tall);
+		_gap = wide + XRES(5);
 	}
 
 	virtual void setText(const char* text)
 	{
+		// std::isspace assert-fails on UTF-8 chars
```

Comment:
How was it not an issue before?

Reply:
Assertions are removed from release builds and I guess no dev encountered a player with non-english chars in the name.

Reply:
Hmm, I suppose, could also be an issue only with one libc implementation.